### ISSUE TITLE
[3.5] edited file name is not saved.

### ISSUE THREAD

**Issue by Donner-Party** [Ultimaker/Cura#4530]
**Application Version**
3.5

**Platform**
Windows 10 x64 build 1809

**Printer**
Geeetech i3W

**Steps to Reproduce**
load an STL
edit the file name
save the file
the name of the saved file does not match the edited file name.

**Actual Results**
the saved file name is the same as before I edited it.

**Expected results**
The name of the saved file should be the same as the file name after I edited it.
I just "upgraded" to 3.5 and did not have this problem in 3.3.

**Additional Information**

![image](https://user-images.githubusercontent.com/44014054/46713386-260ea280-cc0b-11e8-9c20-18980e67a573.png)

[CFFFP_Center Finder (1).zip](https://github.com/Ultimaker/Cura/files/2463123/CFFFP_Center.Finder.1.zip)

[stderr.zip](https://github.com/Ultimaker/Cura/files/2463129/stderr.zip)

**Comment by LipuFei** [comment]
@diegopradogesto Could you try this on Windows? I could not reproduce this on Linux.

**Comment by diegopradogesto** [comment]
No, I can't reproduce it. I tried several things and all of them worked for me. Let's wait until somebody else reported the same issue. Otherwise we will close it.

**Comment by LipuFei** [comment]
Hi @Donner-Party , wonder if this is still a problem for you?

**Comment by Donner-Party** [comment]
Yes, still a problem. Should I remove Cura and re-install?

**Comment by LipuFei** [comment]
@Donner-Party Yes, please try that. And we have a final 3.5 release (not BETA). Can you check if you are using the final release? Thanks

**Comment by pete-ladell** [comment]
I've had this problem since at least v3.3 using win 7. 1st showed when Cura introduced printer models into the file name to be saved. (UM2_FileName) and still with v3.5 Just tried it with win 10 and v 3.2.1 works as it should - saves as edited. & again with 3.5 seems to work as above.
Another difference between 3.2.1 & 3.5  on win 7 I get "Save to File" or "Save to Removable Drive" on win 10 I get "Prepare" before Save to ...

**Comment by ikout** [comment]
I have had this issue as well with the last versions of Cura and it still happens at 3.5 at MacOS...

<img width="868" alt="cura filename issue" src="https://user-images.githubusercontent.com/9326582/46881762-2306f280-ce55-11e8-9dcb-f113cf6fb914.png">

**Comment by ikout** [comment]
I think I found the "**bug**"... If the STL File name has spaces, ie. "Bed Level Testing" and I try to rename it to "Bed Level Testing_v2" the name will be shown at the "filename" field but when I press save due to the spaces at the filename it will save it as the original name....

If the filename has no spaces ie. "Bed_Level_Testing" and I rename it to "Bed_Level_Testing_v5" then it will be saved with the "new" file name:

<img width="868" alt="cura filename issue found" src="https://user-images.githubusercontent.com/9326582/46887565-a5e47900-ce66-11e8-8be8-49a5085892ac.png">

Either handle the filenames with spaces better or put a message that the file cannot be renamed due to spaces at the filename.
